azure-maven-plugin-lib/src/main/resources/bundles/com/microsoft/azure/toolkit/operation.properties

```diff
@@ -8,13 +8,15 @@ user/functionapp.package=generate configuration files and prepare staging direct
 user/functionapp.run=run function app locally
 user/webapp.deploy_app=deploy to Azure Web App with resource creation or updating
 user/webapp.config=generate configuration for web app maven plugin
+user/containerapps.deploy_mojo=deploy Container app to Azure from config
```

Comment:
When referring to a single Container Apps instance, use lower case singular `container app`